lib/matplotlib/spines.pyi

```diff
@@ -63,7 +63,7 @@ class Spine(mpatches.Patch):
 
 class SpinesProxy:
     def __init__(self, spine_dict: dict[str, Spine]) -> None: ...
-    def __getattr__(self, name: str): ...
+    def __getattr__(self, name: str) -> Callable[[Any], None]: ...
```

Comment:
```suggestion
    def __getattr__(self, name: str) -> Callable[[...], None]: ...
```

May work with `Any`, but typing docs indicate `...` as the "any input signature" syntax

Reply:
I originally thought it was only _one_ parameter with any type, as passed to any `.set_*` method, but now I realize the proxy can be for `.set`, which allows multiple parameters, so `...` is better.